.github/scripts/process_commit.py

```diff
@@ -56,11 +84,11 @@ def get_pr_merger_and_labels(pr_number: int) -> Tuple[str, Set[str]]:
     if not pr_number:
         sys.exit(0)
 
-    merger, labels = get_pr_merger_and_labels(pr_number)
-    response = query_pytroch("labels", accept="application/json")
-    response_labels = list(map(lambda x: str(x["name"]), response.json()))
-    primary_labels = set(filter(lambda x: x.startswith(PRIMARY_LEBEL_FILTER), response_labels))
-    is_properly_labeled = bool(primary_labels.intersection(labels) and SECONDARY_LEBELS.intersection(labels))
+    user, labels = get_pr_author_and_labels(pr_number)
+    repo_labels = get_repo_labels()
+
+    primary_labels = set(filter(lambda x: x.startswith(PRIMARY_LABEL_FILTER), repo_labels))
```

Comment:
Why not extract secondary labels in the same fashion rather than have a hardcoded list?

Reply:
True, this is what I initially had in this PR: https://github.com/pytorch/pytorch/pull/71726
However as per discussion it was decided to have the secondary labels be hardcoded